src/test/java/bio/terra/janitor/integration/common/configuration/TestConfiguration.java

```diff
@@ -11,11 +12,30 @@
 public class TestConfiguration {
   private String trackResourceTopicId;
 
+  private String clientSaPath;
```

Comment:
nit: abbreviations are hard to understand later. Ok to spell out ServiceAccount.